donate/templates/pages/core/ways_to_give_page.html

```diff
@@ -0,0 +1,91 @@
+{% extends "pages/base_page.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Ways to Give" %}{% endblock %}
+
+{% block template_name %}app--ways-to-give{% endblock %}
+
+{% block content %}
+
+<div class="banner">
+    <h1>{% trans "Ways to Give" %}</h1>
+    <div><a href="{{ways_to_give_link}}" class="button button--outline button--dark-bg">{% trans "DONATE NOW" context "Button" %}</a></div>
+</div>
+
+<div class="container content">
+    <h2>{% trans "Credit / Debit and PayPal" %}</h2>
+    <p>
+        {% blocktrans trimmed %}
+        <a href="{{ways_to_give_link}}">Go to this link</a> to make a secure online donation right now.
+        {% endblocktrans %}
+    </p>
+
+    <h2 id="wire">{% trans "Bank Transfer / SEPA" %}</h2>
+    <p>{% trans "We accept donations in Euros through Single European Payment Area (SEPA) transfer, and British Pounds through BACS transfer." %}</p>
+    <p>{% trans "Please remember that donations are deductible for U.S. income tax purposes only." %}</p>
+    <p>
+        {% blocktrans with address="331 E. Evelyn Ave, Mountain View, CA 94041 USA" trimmed %}
+        If your bank requests our physical address, it is {{address}}.
+        {% endblocktrans %}
+    </p>
+
+    <h3>{% trans "Euros:" %}</h3>
+    <p>
+        <b>{% trans "Account Currency" %}:</b> {% trans "Euro (EUR)" %}<br>
+        <b>{% trans "Receiving Bank (Pay to)" %}:</b> Standard Chartered Bank<br>
+        <b>{% trans "City & Country/Region" %}:</b> {% trans "Frankfurt, Germany" %}<br>
+        <b>{% trans "SWIFT/BIC Code" %}:</b> SCBLDEFX<br>
+        <b>{% trans "Beneficiary Name (For credit to)" %}:</b> SVB-Mozilla Foundation<br>
+        <b>{% trans "IBAN" %}:</b> DE67512305000500136802
+    </p>
+
+    <h3>{% trans "British Pound Sterling:" %}</h3>
+    <p>
+        <b>{% trans "Account Currency" %}:</b> {% trans "British Pound Sterling (GBP)" %}<br>
+        <b>{% trans "Receiving Bank (Pay to)" %}:</b> National Westminster Bank<br>
+        <b>{% trans "City & Country/Region" %}:</b> {% trans "London, United Kingdom" %}<br>
+        <b>{% trans "Bank Sort Code" %}:</b> 60-00-04<br>
+        <b>{% trans "Account Number (8 digits)" %}:</b> 10017496<br>
+        <b>{% trans "Beneficiary Name (For credit to)" %}:</b> SVB RE Mozilla Foundation
+    </p>
+    <p>
+        <b>{% trans "SWIFT/BIC Code" %}:</b> NWBKGB2L<br>
+        <b>{% trans "IBAN" %}:</b> GB77NWBK60000410017496
```

Comment:
We need to move the colons inside the strings, they need to be localized as well. In French for instance, you need to add a non-breaking space before a colon